backend/src/router.rs

```diff
@@ -136,22 +181,38 @@ impl<V, D, S, B> Deref for ContextRouter<V, D, S, B> {
     }
 }
 
+#[cfg(all(feature = "swagger", debug_assertions))]
+pub trait RouterApiExt<S = (), B = Body, E = Infallible> {
```

Comment:
I see no difference between this trait and the one below. Looks like the separation based on `cfg` is not needed on trait declaration level